### PR TITLE
timeout individual blobs after 15 seconds rather than a minute to limi…

### PR DESCRIPTION
**Pull request by paulgmiller**
…t hiccups, should get retried anyways

### CODE REVIEW

File: Args.cs [hunk 1]
```diff
@@ -25,6 +25,9 @@ public class Args
         [Option]
         public string RelativePath { get; set; } = Environment.GetEnvironmentVariable(RelavePathEnvironmentVariable) 
                                                    ?? "/";
+        
+        [Option]
+        public int BlobTimeoutSeconds { get; set; } = int.Parse(Environment.GetEnvironmentVariable("BlobTimeoutSeconds") ?? "15");
```
Comment (doubleyewdee) [inline Args.cs:30]: If I might suggest a more robust change: wrap this bad boy in int.Min/int.Max to set sane lower and upper bounds so misconfiguration doesn't lead to nasty surprises.

File: VSTSDropProxy.cs [hunk 2]
```diff
@@ -105,6 +108,7 @@ await Policy
                     })
                 .ExecuteAsync(async () =>
                 {
+                    //if we would set the timeout based on the size of the blob which is in the manifest.
```
Comment (doubleyewdee) [inline VSTSDropProxy.cs]: I do not understand the purpose or meaning of the new comment?

**Review by doubleyewdee — CHANGES_REQUESTED** [review]

File: Args.cs (hunk 1 above)
Comment (paulgmiller) [reply Args.cs:30]: Assume you mean wrap the set accessor? I wonder if that gets applied to default values. Guess I can find out.

Comment (paulgmiller) [reply Args.cs:30]: did it in validate.

File: VSTSDropProxy.cs (hunk 2 above)
Comment (paulgmiller) [reply VSTSDropProxy.cs]: slightly better.

**Pull request merged**